android/app/src/main/java/org/mozilla/magnet/permissions/Check.java

```diff
@@ -0,0 +1,44 @@
+package org.mozilla.magnet.permissions;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.content.pm.PackageManager;
+import android.support.v4.content.ContextCompat;
+
+/**
+ * Created by wilsonpage on 09/11/2016.
+ */
+abstract class Check {
```

Comment:
Android studio keeps prompting me to remove redundant scope qualifiers. But happy to be more explicit.